server/models/Notification.js

```diff
@@ -0,0 +1,24 @@
+const mongoose = require('mongoose');
+
+const notificationSchema = new mongoose.Schema({
+    sender: {
+        type: mongoose.Schema.Types.ObjectId,
+        ref: 'user',
+        required: true
+    },
+    receiver: {
+        type: mongoose.Schema.Types.ObjectId,
+        ref: 'user',
+        required: true
+    },
+    notificationType: {
+        type: String,
+        required: true
+    },
+    body: {
```

Comment:
Maybe rename this to something else like `content`? (To avoid confusion when receiving data from req.body)